src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -88,7 +91,36 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
-g.test('it_is_invalid_to_draw_in_a_render_pass_with_missing_bind_groups')
+g.test('set_pipeline_or_bind_group_missing_in_render_pass')
+  .desc('Tests the bind groups required by the pipeline layout are required.')
+  .paramsSubcasesOnly(u =>
+    u
+      .combine('cmd', kDrawCmds)
```

Comment:
optional: When you go to implement this, consider whether it would be easier to merge the render and compute tests - use the `createEncoder` helper to create render and compute pass encoders (can likely choose the encoderType based on which of the 6 `cmd`s is used).

Same comment for some of the other pairs of test as well.

Reply:
Combined the encoders and cmds to params.